src/components/WwwFrame/WwwPage.vue

```diff
@@ -28,6 +28,21 @@ import TheFooter from './TheFooter';
 import TheBasketBar from './TheBasketBar';
 import TheBannerArea from './TheBannerArea';
 
+const hasLentBeforeCookie = 'kvu_lb';
+const hasDepositBeforeCookie = 'kvu_db';
+
+const optimizelyUserDataQuery = gql`query optimizelyUserDataQuery {
+	my {
+		userAccount {
+			id
+			isFirstTimeDepositor
+		}
+		loans {
+			totalCount
+		}
+	}
+}`;
```

Comment:
isFirstTimeDepositor is only true for people with only 1 deposit. Also, the `loans` endpoint is fetching 20 loans by default, which may slow things down. I would suggest checking the `transactions` endpoint to get the total number of deposits and also limit all results:
```graphql
{
  my {
    loans(limit:1) {
      totalCount
    }
    transactions(limit:1, filter:{category:deposit}) {
      totalCount
    }
  }
}
```

Reply:
Nice suggestion, thank you!

Reply:
Done!